TS06-Facets-And-NDVI-Plotting-In-R.Rmd

```diff
@@ -127,95 +106,347 @@ library(dplyr)  #for subsetting by season
 #setwd("working-dir-path-here")
 
 #daily HARV met data, 2009-2011
-harMetDaily.09.11 <- read.csv(file="AtmosData/HARV/Met_HARV_Daily_2009_2011.csv",
+harMetDaily.09.11 <- read.csv(file="NEON-DS-Met-Time-Series/HARV/FisherTower-Met/Met_HARV_Daily_2009_2011.csv",
                      stringsAsFactors = FALSE)
 
 #covert date to POSIXct date-time class
 harMetDaily.09.11$date <- as.POSIXct(harMetDaily.09.11$date)
 
 #monthly HARV temperature data, 2009-2011
-harTemp.monthly.09.11<-read.csv(file="AtmosData/HARV/Temp_HARV_Monthly_09_11.csv",
-                                stringsAsFactors=FALSE)
+#harTemp.monthly.09.11<-read.csv(file="NEON-DS-Met-Time-Series/HARV/FisherTower-Met/Temp_HARV_Monthly_09_11.csv",
+#                                stringsAsFactors=FALSE)
 
 #convert datetime from chr to datetime class & rename date for clarification
-harTemp.monthly.09.11$date <- as.POSIXct(harTemp.monthly.09.11$datetime)
+#harTemp.monthly.09.11$date <- as.POSIXct(harTemp.monthly.09.11$datetime)
+```
+
+##GGPLOT Facets
+
+Facets allow us to plot subsets of data in one cleanly organized panel. We use
+ `facet_grid()` to create a plot of a particular variable subsetted by a particular
+ "group".
+ 
+Let's plot air temperature as we did previously. We will name the `ggplot`
+object `AirTempDaily`.
+
+```{r plot-airt }
+
+AirTempDaily <- ggplot(harMetDaily.09.11, aes(date, airt)) +
+           geom_point() +
+           ggtitle("Daily Air Temperature NEON Harvard Forest\n 2009-2011") +
+            xlab("Date") + ylab("Temperature (C)") +
+            scale_x_datetime(labels=date_format ("%m-%y"))+
+           theme(plot.title = element_text(lineheight=.8, face="bold",
+                 size = 20)) +
+           theme(text = element_text(size=18))
+
+AirTempDaily
+```
+ 
+This plot tells us a lot about the annual increase and decrease of temperature
+at the NEON Harvard Forest field site. However, what is we wanted to plot each
+year's worth of data individually?
+
+If we use the `facet()` element in `ggplot`, we can create facets or a panel of 
+plots that are grouped by a particular category or time period. To create a 
+a plot for each year, we will first need a year column in our data to use a subset 
+factor. We created a year column did this in the 
+[{{site.baseurl}}R/Time-Series-Subset-dplyr/](DPLYR subset timeseries lesson)
```

Comment:
i can't see the code above to understand what you want to be adjusted. to show code, but your code in brackets (```)
```
   <p>here is some code</p>
   ### here is some code
```